subprojects/build-cache-packaging/src/main/java/org/gradle/caching/internal/command/BuildCacheCommandFactory.java

```diff
@@ -93,7 +93,7 @@ public BuildCacheKey getKey() {
         public BuildCacheLoadCommand.Result<LoadMetadata> load(InputStream input) throws IOException {
             BuildCacheEntryPacker.UnpackResult unpackResult = packer.unpack(entity, input, originMetadataFactory.createReader(entity));
             ImmutableSortedMap<String, CurrentFileCollectionFingerprint> snapshots = snapshotUnpackedData(unpackResult.getSnapshots());
-            LOGGER.info("Unpacked trees for {} from cache.", entity.getDisplayName());
+            LOGGER.debug("Unpacked trees for {} from cache.", entity.getDisplayName());
```

Comment:
I think the info logs should at least show if the task was loaded from cache or stored in the cache. If we move the two logging levels to `debug`, then no interaction with the build cache is shown at all.

Reply:
I've reverted this.